app/src/main/java/org/schabi/newpipe/local/feed/FeedFragment.kt

```diff
@@ -311,6 +410,33 @@ class FeedFragment : BaseListFragment<FeedState, Unit>() {
         return true
     }
 
+    ///////////////////////////////////////////////////////////////////////////
+    // Grid Mode
+    ///////////////////////////////////////////////////////////////////////////
+
+    // TODO: Move these out of this class, as it can be reused
```

Comment:
Maybe move these two function to the `ThemeHelper` utils class?

Reply:
I'll leave it for another pull request.